storage-bigtable/src/confirmed_block.proto

```diff
@@ -73,6 +73,10 @@ message Reward {
     RewardType reward_type = 4;
 }
 
+message Rewards {
```

Comment:
Thoughts on creating a `solana-proto` crate that both `solana-ledger` and `solana-storage-bigtable` would depend on?
It was easy not to, since `solana-ledger` already depends on `solana-storage-bigtable`

Reply:
`solana-ledger` depending on `solana-storage-bigtable` for rocksdb storage doesn't feel great.

`solana-ledger-proto` or `solana-storage-proto` maybe?  `solana-proto` feels too generic.   Then `solana-*-capnproto` slides in nicely later

Reply:
Incidentally, I'm happy to give capnproto a go for this, but as you pointed out, the rocksdb and bigtable reuse seems nice...

Reply:
IMO let's go for reuse here.  If capnproto works out for nicely for program instruction/account serde then we can potentially use it here as well if we're really concerned about the additional cost that protobuf brings

Reply:
@mvines how does this look?